src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/calculatereleasedatesapi/service/FindHistoricReleaseDateServiceTest.kt

```diff
@@ -131,7 +131,7 @@ class FindHistoricReleaseDateServiceTest {
 
     val exception = assertThrows<UnsupportedCalculationException> { service.findReleaseDate(prisonerId, emptyList(), listOf(sentence), calculateAt, charges) }
 
-    assertThat(exception.message).isEqualTo("The release date is before the calculation date.")
+    assertThat(exception.message).isEqualTo("The release date 2019-12-30, from calculations [1] is before the calculation date 2020-01-06T00:00.")
```

Comment:
This text comes out a bit unreadably? Might be good to format the second date consistently?

Reply:
The first one is a localdate, the second is a localdatetime. This message is never shown to users. Its just for debugging some test cases on preprod.